pywbem/mof_compiler.py

```diff
@@ -1915,6 +1910,9 @@ class MOFWBEMConnection(BaseRepositoryConnection):
     This class implements the
     :class:`~pywbem.BaseRepositoryConnection` interface.
 
+    This implementation does not force paths on instances and just appends
+    each new createinstance to the instance repository
```

Comment:
On "each new createinstance": Change to "each new instance".

Reply:
Agreed. DONE